### DISCUSSION TITLE
Help with QRControl on Android.

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#298] [General]
QRControl is getting negative reviews as some phones aren't correctly honoring the screen brightness command.  Unfortunately those who leaving one star reviews, are rarely interested in helping to solve the problem. I'm looking for users of QRControl on Android who have this issue, but willing to help find a solution.

This Auto Brightness control is not working on some phones:
![image](https://user-images.githubusercontent.com/10856084/230734540-fee1a274-4f9d-460d-83fe-cf7286017791.png)

With it enabled the app should be a maximum brightness (like it is on my devices), but some users report that the screen goes very dark. I'm trying to determine if this is a database issue (old data setting the wrong brightness) or device compatibility issue (setting the wrong level for those devices.)

**Comment by dnewman-gpsw** [comment]
Bump, I really need some external help on this.  

Here is a beta APK for 1.8.3 https://www.dropbox.com/t/57G84buHCmc4AxyD 

Please report back if the Auto brightness control works or not.

**Reply by RibLaiva** [reply]
Hello. Where can I download Android apk QRControl ? Google play store seems deleted it.

**Comment by dnewman-gpsw** [comment]
It is live again. https://play.google.com/store/apps/details?id=com.miscdata.qrcontrol

**Comment by Artushfoto** [comment]
On the Galaxy S22 and Galaxy S25 Ultra with the latest system updates, automatic brightness control works perfectly. I haven't noticed any bugs.
